google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -269,14 +270,15 @@ private LogEntry logEntryFor(ILoggingEvent e) {
 
     Level level = e.getLevel();
     LogEntry.Builder builder =
-        LogEntry.newBuilder(Payload.StringPayload.of(payload.toString().trim()))
```

Comment:
@athakor Can you please tell me why exactly are we changing the payload message with error message? Does payload contains full stacktrace?